src/static/css/pad/layout.css

```
@@ -4,6 +4,14 @@ html, body {
   margin: 0;
   padding: 0;
 }
+
+html.timeslider {
```

To be 100% consistent with what it was before commit 470f40d7db01b68856edf47ac7a5befd50b50340:

```suggestion
html.timeslider, html.timeslider body {
```